specs/rollup-node.md

```diff
@@ -93,13 +93,13 @@ Note that the sequencing windows overlap.
 The rollup reads the following data from the [sequencing window][g-sequencing-window]:
 
 - Of the *first* block in the window only:
+  - L1 log entries emitted for [user deposits][g-deposits], augmented with a [sourceHash](./deposits.md#).
+- Of each block in the window:
```

Comment:
The report says: 
> Some portions of documentation state that the L1 attributes deposited transaction is included only in the first block of a sequencing window, while other portions of the documentation state that it is included in every L2 block. (The latter is the correct behavior.)

This change doesn't quite seem to reflect what they describe. It refers to reading data from L1, not posting to L2. Is what they describe as the actual behavior somehow implied by this change?

Reply:
gah, you're right. This fix was wrong. The sequencing window is not the same as the L2 epoch (though one sequencing window does correspond to a sequencing window.

Reply:
This fix request first links to the line "Of the first block in the window only". The window in question is the sequencing window, the set of L1 blocks from which an L2 epoch is derived. It is correct that the L1 Attributes are read from the first block of the L1 Sequencing window. The second link points to the fact that there must be a L1 Attributes Transaction for every L2 block. Given the first point is about L1 blocks, and the second is about L2 blocks, there is no inconsistency.